larq/quantizers.py

```diff
@@ -156,10 +153,10 @@ def ste_sign(x, clip_value=1.0):
       Activations Constrained to +1 or -1](http://arxiv.org/abs/1602.02830)
     """
 
-    if clip_value is not None:
-        x = tf.clip_by_value(x, -clip_value, clip_value)
+    def grad(dy):
+        return _clipped_gradient(x, dy, clip_value)
 
-    return _binarize_with_identity_grad(x)
+    return math.sign(x), grad
```

Comment:
We could also use a lambda function here, but I think a explicit `grad` function is more readable for people with less experience with `ﬂtf.custom_gradient`.